governance.md

```diff
@@ -210,7 +207,6 @@ All contributors must sign the CNCF CLA, as described [here](CLA.md).
 [sig governance]: /committee-steering/governance/sig-governance.md
 [owners]: /community-membership.md#subproject-owner
 [sig charter process]: /committee-steering/governance/README.md
-[short template]: /committee-steering/governance/sig-governance-template-short.md
```

Comment:
was it changed to `/committee-steering/governance/sig-charter-template.md`?

Reply:
@idealhack thanks for looking into this! 
I don't think that it was changed to `/committee-steering/governance/sig-charter-template.md`. I don't know the exact history of those two documents but currently in the `[sig charter process]` document, the very first link points to the `/committee-steering/governance/sig-charter-template.md` and its content seems to have the same goal as the `sig-governance-temaplte-short.md` (Historical snapshot can be found here https://github.com/kubernetes/community/commit/68658bff6ef05ed2cceda5e97efaee8d1f093233#diff-a7656e2a5ad72df5591ce73122842692 ).

Reply:
@Disper thanks for the link to the historical snapshot! Based on it, looks like the new doc should be https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance-requirements.md.